app/controllers/historic_appointments_controller.rb

```diff
@@ -0,0 +1,6 @@
+class HistoricAppointmentsController < ApplicationController
```

Comment:
Following on the pattern from the previous two Whitehall imports ([Past Foreign Secretaries](https://github.com/alphagov/collections/pull/3113) and [Past Chancellors of the Exchequer](https://github.com/alphagov/collections/pull/3118)), could we rename controller/models/views to PastPrimeMinisters?

Or, I'm wondering if this generic name symbolises the first step in a plan to do some refactoring? ie there's currently a lot of duplication, [PastChancellors](https://github.com/alphagov/collections/blob/main/app/models/past_chancellors.rb) and [PastForeignSecretaries](https://github.com/alphagov/collections/blob/main/app/models/past_foreign_secretaries.rb) are identical. And they contain code duplicated again in this new [HistoricAppointment](https://github.com/alphagov/collections/blob/e238bf13e3bffb8554cf0a763c0a9a7c122f9d58/app/models/historic_appointment.rb) model. 

I'm wondering why we don't have a generic historic appointment (as seems to be the case in Whitehall where this code is coming from), with PastForeignSecretaries, PastChancellors and PastPrimeMinisters all inheriting from the parent class (and possibly parent controller too). 

Or is there some reason they all need to be separate that I'm missing?

Sorry to be a pain - but now this is a third import it does feel like we need to refactor a bit.

Reply:
or a module with the same methods to include instead